### PR TITLE
merge from main to teams branch

### PR DESCRIPTION
**Pull request by iaulakh**
## Purpose

* merge from main to teams branch

## Does this introduce a breaking change?

```
[x] Yes
[ ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[x] Other... Please describe:
```

### CODE REVIEW

**Comment by iaulakh** [comment]
@Gemakk @lpx1989 I noticed a strange dialog when I end call on this merge. may you please test?

**Comment by Gemakk** [comment]
Do you mean this? 
image

**Comment by iaulakh** [comment]
> Do you mean this?
> image

yes this dialog appears when I click leave call

File: AzureCalling/app/src/main/res/layout-land/activity_call.xml [hunk 1]
```diff
@@ -31,27 +31,32 @@
...
-            android:layout_marginBottom="20dp"
-            app:layout_constraintBottom_toBottomOf="parent"
-            app:layout_constraintEnd_toEndOf="parent">
-
+            android:id="@+id/relativeLayout"
+            android:layout_width="match_parent"
+            android:layout_height="match_parent">
+
+            <include layout="@layout/participant_header_notification"/>
+            <include layout="@layout/floating_header_area" />
```
Comment (Gemakk) [inline AzureCalling/app/src/main/res/layout-land/activity_call.xml:40]: Looks like this element no longer shows up during Teams calls.  Specifically, the recording notifications.

**Review by Gemakk — CHANGES_REQUESTED** [review]
I'm unable to receive Recording notifications from Teams calls

**Comment by iaulakh** [comment]
fixed merge issue, now we can see notification but :)
created bug @Gemakk : 
https://skype.visualstudio.com/SPOOL/_sprints/taskboard/Mobile%20UI/SPOOL/CY2021-Q2/Sprint%2038%20(Ends%20June%2013)?workitem=2476001

**Review by Gemakk — APPROVED** [review]
Tested

**Review by lpeixin — APPROVED** [review]
Tested.

**Pull request merged**